src/v2/Apps/Categories/Components/GeneFamily.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Link, Spacer, Text, themeProps } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { GeneFamily_geneFamily } from "v2/__generated__/GeneFamily_geneFamily.graphql"
+import { sortBy } from "lodash"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+interface GeneFamilyProps {
+  geneFamily: GeneFamily_geneFamily
+}
+
+type Genes = GeneFamily_geneFamily["genes"]
+
+const alphabetizeGenes = (genes: Genes): Genes =>
+  sortBy(genes, gene => gene?.displayName || gene?.name)
+
+export const GeneFamily: React.FC<GeneFamilyProps> = props => {
+  const { geneFamily } = props
+  const { name, genes } = geneFamily
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+
+  if (!genes) {
+    return null
+  }
+
+  const publishedGenes = genes.filter(g => !!g && g.isPublished)
+  const sortedGenes = alphabetizeGenes(publishedGenes)
+
+  return (
+    <Box id={`jump--${geneFamily.slug}`}>
+      <Text variant="xl">{name}</Text>
+      <Spacer mt={4} />
+      <ul style={{ columnCount: isMobile ? 1 : 3 }}>
+        {sortedGenes?.map(gene => {
+          return (
+            <Box key={gene?.id}>
+              <Link href={`/gene/${gene?.slug}`} noUnderline={true}>
```

Comment:
Since we're now in a single page app context, we'll always want to use `<RouterLink to={some-url} textDecoration='none'>` over `Link` from palette. That will allow us to avoid hard jumps between pages and get the smooth page loader at the top.